tests/ludwig/hyperopt/test_hyperopt.py

```diff
@@ -82,10 +82,12 @@ def test_grid_search_more_than_one_sample():
     with pytest.warns(RuntimeWarning):
```

Comment:
nit: Could you add a short docstring explaining the test?

Reply:
Done!